geode-core/src/main/java/org/apache/geode/internal/cache/partitioned/rebalance/model/RefusalReason.java

```diff
@@ -42,6 +48,11 @@ public String formatMessage(Member target, Bucket bucket) {
       case CRITICAL_HEAP:
         return "Target member " + target.getMemberId()
             + " has reached its critical heap percentage, and cannot accept more data";
+      case DIFFERENT_ZONE:
```

Comment:
All of these other `RefusalReasons` are refusing to create a bucket. I don't think DIFFERENT_ZONE belongs in this enum. Based on my ealier comment, I think you can just remove it entirely since we don't use it.